docs/operations/replication.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "How To Use Volume Replication"
+linkTitle: How To Use Volume Replication
+---
+
+## Overview
+
+> 💡 For more information on Ondat's Replication feature, review the  [Replication](/docs/concepts/replication)  feature page.
+
+### Example - Enable Volume Replication Through a `PersistentVolumeClaim` Definition
+
+The following guidance below will demonstrates how to use Ondat's Volume Replication  through a  `PersistentVolumeClaim`  (PVC) definition.
+-   The instructions will enable volume replication on a PVC with the label »  `storageos.com/replicas=1` - which will create `1` master volume and `1` replica volume respectively.
+
+    > 💡 Labels can be applied to a PVC directly, or indirectly by adding them as parameters on a `StorageClass`.
+
+1.  Create a custom `PersistentVolumeClaim` named `pvc-replicated` and ensure that you add the following label >> `storageos.com/replicas=1` to the manifest.
+
+    ```yaml
+    # Create a "pvc-replicated" PVC that has a replica volume count of 1.
+    cat <<EOF | kubectl create --filename -
+    apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: pvc-replicated
+      labels:
+        storageos.com/replicas: "1"                  # Replica volume count of 1
+    spec:
+      storageClassName: storageos
+      accessModes:
+        - ReadWriteOnce
+      resources:
+        requests:
+          storage: 5Gi
+    EOF
+    ```
+
+1.  Once the PVC resource has been successfully created, review and confirm that the  `storageos.com/replicas=1`, label has been applied.
+
+    ```bash
+    # Get the label applied to the "pvc-replicated" PVC.
+    kubectl get pvc pvc-replicated --output=wide --show-labels --namespace=default
+
+    NAME             STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE   VOLUMEMODE   LABELS
+    pvc-replicated   Bound    pvc-7fed5a56-42b2-4fe3-bcab-e31c97931b8c   5Gi        RWO            storageos      26s   Filesystem   storageos.com/replicas=1```
+    ```
+
+1. To review and confirm that Ondat has successfully provisioned `1` master volume and `1` replica volume as defined in the PVC manifest earlier - deploy and run the  [Ondat CLI utility as a deployment](https://docs.ondat.io/docs/reference/cli/#run-the-cli-as-a-deployment-in-your-cluster)  first, so that you can interact and manage Ondat through  `kubectl`. Once deployed, obtain the Ondat CLI utility pod name for later reference.
```

Comment:
remove extra space before 'first, so that you can...'

Reply:
remove extra space before `kubectl`. Once...

Reply:
fixed